packages/dialog/src/css/index.css

```diff
@@ -0,0 +1,128 @@
+@keyframes psds-dialog__keyframes__fade {
+  100% {
+    transform: translateY(0);
+    opacity: 1;
+  }
+}
+.psds-dialog {
+  animation: psds-dialog__keyframes__fade var(--ps-motion-speed-fast) ease-out
+    forwards;
+  background: var(--ps-colors-white);
+  border-radius: 2px;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.5);
+  display: inline-flex;
+  opacity: 0;
+  position: relative;
+  transform: translateY(var(--ps-layout-spacing-xsmall));
+}
+.psds-dialog:focus {
+  outline: 0;
+}
+.psds-dialog__content {
+  color: var(--ps-colors-text-icon-high-on-light);
+  font-size: var(--ps-type-font-size-200);
+  font-weight: var(--ps-type-font-weight-500);
+  line-height: var(--ps-type-line-height-tight);
+  padding: var(--ps-layout-spacing-large);
+  flex: 1 1 auto;
+  overflow-y: auto;
+}
+.psds-dialog--w-tail:after {
+  background-color: var(--ps-colors-white);
+  content: ' ';
+  display: block;
+  height: 14px;
+  line-height: 0;
+  position: absolute;
+  transform: rotate(45deg);
+  white-space: pre;
+  width: 14px;
+}
+.psds-dialog--tailposition-topleft:after {
```

Comment:
Should we  dash case these tail position classes. I don't feel strongly on it but it would be nice to decided on when we want to treat things as whole words versus hyphenated ones.

Reply:
Great question.

What I think we could do: (diff from current practice, as I paused to think about your question)

1. Insert dashes everywhere there are separate works (where there once were Capitals)
2. dashify in code the js vars that concat into selectors

For example:
```
import { dashify } from '......-util'
classNames(`psds-dialog--tail-position-${dashify(tailPosition)}`)
```

Today I'm converting like this usually:

```
classNames(`psds-dialog--tail-position-${tailPosition.toLowerCase()}`)
```

And in this case even missed a dash:

```
classNames(`psds-dialog--tailposition-${tailPosition.toLowerCase()}`)
```

So I think there is some refactoring work to do to use a util#dashify instead of toLowerCase. I'll plan to do that in my next PR.

Reply:
Talked together again. This is the more-full standard:

1. Dashify whole words (eg, `tail-position`)
2. Acronomyns are not dashed (eg, `svg`, `csv`)
3. Abbreviations are not dashed (eg, `xsmall`, `xxlarge`)